internal/pkg_npm/packager.js

```diff
@@ -186,13 +186,30 @@ function main(args) {
    * @returns {string}
    */
   function getOwningPackageRelativeOutPath(file) {
+    // below isn't indented correctly so that that we can
```

Comment:
Would the extra whitespace even cause a merge conflict? I thought git was permissive to changes that are only whitespace.

Reply:
If we do stick with this, it's a little verbose, maybe just:
We've intentionally modified the indentation below to avoid merge conflicts

Reply:
We are unlikely to ever pull down new code from upstream here, so we shouldn't need to skew indentation for that reason.